packages/publisher/electron-release-server/src/PublisherERS.ts

```diff
@@ -17,6 +17,17 @@ interface ERSVersion {
   assets: { name: string; }[];
 }
 
+const fetchAndCheckStatus = async (
+  url: RequestInfo,
+  init?: RequestInit,
+): Promise<Response> => {
+  const result = await fetch(url, init);
+  if (result.ok) { // res.status >= 200 && res.status < 300
+    return result;
+  }
+  throw new Error(`ERS publish failed with status code: ${result.statusText} ${result.url}`);
```

Comment:
Perhaps for clarity, add some parentheses:

```suggestion
  throw new Error(`ERS publish failed with status code: ${result.statusText} (${result.url})`);
```